### ISSUE TITLE
Help

### ISSUE THREAD

**Issue by Urbaee** [open-wa/wa-automate-nodejs#1282]
How to downgrade and keep the version i installed in package.json?
i use this npm i --save @open-wa/wa-automate@3.2.0

And that's change the version in package.json
But when i push in github and try git clone
And i running command npm install 
They install the new version of @open-wa
I mean, i want they install the version before i push in github but why they install the new version?

Please help

**Comment by smashah** [comment]
@Urbaee please share your package.JSON file

**Comment by Urbaee** [comment]
> @Urbaee please share your package.JSON file

![Screenshot_20210201-081129.png](https://user-images.githubusercontent.com/75126993/106404363-2333bd80-6465-11eb-8793-086d7d0c8a7b.png)

Look, that's the version, but when i try git clone and npm install, they install the new version

You can try git clone my repo
That's will install the new version

My repo : https://github.com/Urbaee/whatsapp-bot2

**Comment by smashah** [comment]
@Urbaee 

If you want to stick to one specific version, you have to remove the `^`.

This will make sure the dependency that is installed is greater than 3.2.0:

```javascript
"@open-wa/wa-automate": "^3.2.0",
```

whereas, this will result in the version staying at 3.2.0:

```javascript
"@open-wa/wa-automate": "3.2.0",
```

Also, why exactly do you want your users to be stuck at 3.2.0? For this library it is always best to keep up to date with the latest version.

**Comment by Urbaee** [comment]
> @Urbaee 
> 
> If you want to stick to one specific version, you have to remove the `^`.
> 
> This will make sure the dependency that is installed is greater than 3.2.0:
> 
> ```javascript
> "@open-wa/wa-automate": "^3.2.0",
> ```
> 
> whereas, this will result in the version staying at 3.2.0:
> 
> ```javascript
> "@open-wa/wa-automate": "3.2.0",
> ```
> 
> Also, why exactly do you want your users to be stuck at 3.2.0? For this library it is always best to keep up to date with the latest version.

My code is not supported for 3.2.1 for sticker
It's get "failed with status code"

**Comment by Urbaee** [comment]
![Screenshot_20210201-102718.png](https://user-images.githubusercontent.com/75126993/106411537-1a98b280-6478-11eb-8b06-a9f846525297.png)

**Comment by Urbaee** [comment]
> @Urbaee 
> 
> If you want to stick to one specific version, you have to remove the `^`.
> 
> This will make sure the dependency that is installed is greater than 3.2.0:
> 
> ```javascript
> "@open-wa/wa-automate": "^3.2.0",
> ```
> 
> whereas, this will result in the version staying at 3.2.0:
> 
> ```javascript
> "@open-wa/wa-automate": "3.2.0",
> ```
> 
> Also, why exactly do you want your users to be stuck at 3.2.0? For this library it is always best to keep up to date with the latest version.

Also did the version 3.2.0 not have function ```sendStickerAsReply``` ?

**Comment by smashah** [comment]
> > @Urbaee
> > If you want to stick to one specific version, you have to remove the `^`.
> > This will make sure the dependency that is installed is greater than 3.2.0:
> > ```js
> > "@open-wa/wa-automate": "^3.2.0",
> > ```
> > 
> > 
> > whereas, this will result in the version staying at 3.2.0:
> > ```js
> > "@open-wa/wa-automate": "3.2.0",
> > ```
> > 
> > 
> > Also, why exactly do you want your users to be stuck at 3.2.0? For this library it is always best to keep up to date with the latest version.
> 
> My code is not supported for 3.2.1 for sticker
> It's get "failed with status code"

well this is the main issue then. Will be fixed momentarily

**Comment by Urbaee** [comment]
> > > @Urbaee
> > > If you want to stick to one specific version, you have to remove the `^`.
> > > This will make sure the dependency that is installed is greater than 3.2.0:
> > > ```js
> > > "@open-wa/wa-automate": "^3.2.0",
> > > ```
> > > 
> > > 
> > > whereas, this will result in the version staying at 3.2.0:
> > > ```js
> > > "@open-wa/wa-automate": "3.2.0",
> > > ```
> > > 
> > > 
> > > Also, why exactly do you want your users to be stuck at 3.2.0? For this library it is always best to keep up to date with the latest version.
> > 
> > My code is not supported for 3.2.1 for sticker
> > It's get "failed with status code"
> 
> well this is the main issue then. Will be fixed momentarily

Okay i'll be waiting

**Comment by smashah** [comment]
@Urbaee fixed in 3.2.2

thanks
